app/lib/features/settings/widgets/settings_menu.dart

```diff
@@ -14,38 +14,49 @@ class SettingsMenu extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final currentRoute = ref.watch(currentRoutingLocation);
+    final size = MediaQuery.of(context).size;
     bool isSelected(Routes route) {
       debugPrint(
         '${route.route} $currentRoute, ${currentRoute == route.route}',
       );
       return currentRoute == route.route;
     }
 
-    return SettingsList(
-      sections: [
-        SettingsSection(
-          title: const Text('App Settings'),
-          tiles: [
-            CustomSettingsTile(
-              child: ListTile(
-                title: const Text('Labs'),
-                subtitle: const Text('Experimental Acter features'),
-                leading: const Icon(Atlas.lab_appliance_thin),
-                selected: isSelected(Routes.settingsLabs),
-                onTap: () => context.pushNamed(Routes.settingsLabs.name),
-              ),
-            ),
-            CustomSettingsTile(
-              child: ListTile(
-                title: const Text('Info'),
-                leading: const Icon(Atlas.info_circle_thin),
-                selected: isSelected(Routes.info),
-                onTap: () => context.pushNamed(Routes.info.name),
-              ),
-            ),
-          ],
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'App Settings',
+          style: Theme.of(context).textTheme.titleLarge,
         ),
-      ],
+        centerTitle: false,
+      ),
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          ListTile(
```

Comment:
not sure I like this. Why did you remove the `SettingsList` and `SettingsSection` here? Those allow for nice separation and though aren't doing much yet, are meant to be there for making it easier to extend later. I'd rather we kept the SettingsList and SettingsSections ...

Reply:
The ```SettingsList``` makes the list appear centrally when having enough space and not split view .i.e. on larger screen width. This was looking not so good from interface perspective as we want to keep list aligned to the start whether the splitting is triggered or not.

Reply:
I see ... okay then.